blueprints/deno-kv-litefs.html.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Deno KV with LiteFS Cloud
+layout: docs
+sitemap: false
+nav: firecracker
+author: jesse
+date: 2024-05-28
+---
+
+## Problem
+
+If you're porting an app from Deno Deploy to Fly.io, you may hit a bit of a snag: if it was built with [Deno KV](https://deno.com/blog/kv), a _traditionally_ FoundationDB backed K/V store made for Deno Deploy, it may seem like you're stuck because there's no obvious way to easily run Deno KV at scale on Fly.io. You _could_ use [denoland/denokv](https://github.com/denoland/denokv) on Fly.io, but due to an alignment of the stars there happens to be an easier way.
```

Comment:
Suggest removing the "stuck" part, since we already say "snag":
```suggestion
If you're porting an app from Deno Deploy to Fly.io, you may hit a bit of a snag: if it was built with [Deno KV](https://deno.com/blog/kv), a _traditionally_ FoundationDB backed K/V store made for Deno Deploy, there's no obvious way to easily run Deno KV at scale on Fly.io. You _could_ use [denoland/denokv](https://github.com/denoland/denokv) on Fly.io, but due to an alignment of the stars there happens to be an easier way.
```